src/i18n/de/annotation-image.json

```diff
@@ -1,3 +1,9 @@
 {
-  "Anonymous": "Anonym" 
+  "Anonymous": "Anonym",
```

Comment:
Wait, there's actually an `annotation-common.json` file that get's included in both (text and image) annotation label sets! I.e. this should better go in there. No need for duplication.

Reply:
Had no idea ;-) Moved to common.